code/modules/materials/Mat_FabRecipes.dm

```diff
@@ -75,10 +75,18 @@
 			var/obj/item/clothing/head/helmet/space/custom/helmet = new()
 			suit.set_loc(getOutputLocation(owner))
 			helmet.set_loc(getOutputLocation(owner))
-			var/obj/item/fab = getObjectByPartName("Fabric")
+			var/obj/item/fabr = getObjectByPartName("Fabric")
 			var/obj/item/vis = getObjectByPartName("Visor")
-			suit.setMaterial(fab.material)
-			helmet.setMaterial(vis.material)
+			var/obj/item/renf = getObjectByPartName("Reinforcement")
+			suit.setMaterial(fabr.material)// suit stuff
+			suit.setupSuitProp(fabr.material, renf.material)
+			suit.setsuitname(fabr, renf)
+			if (fabr?.material && vis?.material) // helmet stuff
```

Comment:
I don't quite get why the helmet and only the helmet has this check?

Reply:
that's probably leftover code from when I copied it off of the arrow; removed